hevc/pps_test.go

```diff
@@ -0,0 +1,32 @@
+package hevc
+
+import (
+	"encoding/hex"
+	"github.com/go-test/deep"
+	"testing"
+)
+
+const pps1 = "4401c0f7c0cc90"
+
+func TestPPSParser(t *testing.T) {
+	byteData, _ := hex.DecodeString(pps1)
```

Comment:
It's good to have a test. I understand that is hard to find a test for most of the parameters, but we should aim for adding more pps data over time. For that reason, it may be good to have a table-driven test, but I think it can wait for later.